src/SDKs/ServiceFabric/Management.ServiceFabric/Properties/AssemblyInfo.cs

```diff
@@ -25,5 +25,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.0.1.0")]
```

Comment:
@chingchenmsft no need to update AssemblyVersion, keep it was it was earlier 1.0.0.0 only change the assemblyFileVersion